index.html

```diff
@@ -0,0 +1,106 @@
+<html>
+    <head>
+        <link rel="stylesheet" href="style.css">
+    </head>
+    <body>
+
+        <main id="main">
```

Comment:
It is better to use < section id="main" >

Reply:
- [ ] Well done on using inline comments and giving a polite suggestion. 👍 
- Kindly give a reason `why` it is `better to use < section id="main" >`